test/bdd/fixtures/demo/docker-compose-sidetree-fabric.yml

```diff
@@ -184,7 +184,7 @@ services:
       # metrics config
       - CORE_METRICS_PROVIDER=prometheus
       - CORE_OPERATIONS_LISTENADDRESS=0.0.0.0:8080
-      - CORE_LEDGER_ROLES=endorser,committer,sidetree-resolver,sidetree-batch-writer,sidetree-monitor
+      - CORE_LEDGER_ROLES=endorser,committer,sidetree-resolver,sidetree-batch-writer,sidetree-observer-standby
```

Comment:
Should be sidetree-observer for all peers if not running in cluster mode. In cluster mode there will be one sidetree-observer and zero or more sidetree-observer-standby.

Reply:
@bstasyszyn  Done